webapp/components/form/input.js

```diff
@@ -30,7 +30,7 @@ const InputComponent = (props, ref) => {
           mask={mask}
           className="form-input"
           aria-disabled={disabled}
-          isControlled={true}
+          isControlled={false}
           value={value}
           onChange={(e, { value }) => onChangeValue(value)}
```

Comment:
Should the naming convention be in these cases `event`?
Although, now when not used maybe replace it with `_`